docs/java-rest/high-level/indices/shrink_index.asciidoc

```diff
@@ -47,12 +47,21 @@ returns a response, as an `int`
 <2> The number of active shard copies to wait for before the shrink index API
 returns a response, as an `ActiveShardCount`
 
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests-file}[{api}-request-copySettings]
+--------------------------------------------------
+<1> Use `true` to copy the settings from the source index to the target
+index. This cannot be `false`. If this method is not used, default behavior is not to copy.
```

Comment:
Can you wrap to 80 columns?

Reply:
Fixed in https://github.com/elastic/elasticsearch/pull/39752/commits/b20c245d526abdad6902a2d467b1e1ef9c30ab76